presidio-analyzer/presidio_analyzer/nlp_engine/nlp_artifacts.py

```diff
@@ -18,15 +26,17 @@ def __init__(
         tokens: Doc,
         tokens_indices: List[int],
         lemmas: List[str],
-        nlp_engine,  # noqa ANN001
+        nlp_engine: "NlpEngine",  # noqa F821
```

Comment:
Also checking that "NlpEngine" here is supposed to be a string

Reply:
Good catch, fixing.

Reply:
Just checked again, it has to be in quotes, otherwise we have a circular import as the `NlpEngine` class holds `NlpArtifacts`